app/components/ContentBox/box.css

```diff
@@ -52,3 +52,15 @@
   padding: 0;
   padding-right: var(--spacing-8);
 }
+
+@media (max-width: 600px) {
+  .main-container-box-table {
+    flex-direction: column;
+    height: 550px;
+    margin-bottom: var(--spacing-40);
```

Comment:
Looks good but remember that --spacing-40 turns into I think 24px on mobile, so hope you were trying to make this appear 24px!
<img width="169" alt="image" src="https://github.com/StampyAI/stampy-ui/assets/825910/627d8070-2b35-4bdd-b69b-5041d9016d3c">